pkg/dataflatten/string_delimited.go

```diff
@@ -0,0 +1,69 @@
+package dataflatten
+
+import (
+	"bufio"
+	"bytes"
+	"strconv"
+	"strings"
+)
+
+type dataFunc func(data []byte, opts ...FlattenOpts) ([]Row, error)
+
+// StringDelimitedUnseparatedFunc returns a function that conforms to the
+// interface expected by dataflattentable.Table's execDataFunc property.
+// properties are grouped into a single record based on 'duplicate key'
+// strategy: If a key/value pair is encountered, and the record being built
+// already has a value for that key, then that record is considered 'complete'.
+// The record is stored in the collection, and a new record is started. This
+// strategy is only suitable if the data output does not exclude k/v pairs with
```

Comment:
That's kind of apples-to-oranges, the older bssid detection was really just a way to label grouped properties in the generated SQL results, and it depended on _already knowing_ how properties were grouped, whereas this is a method for determining _how_ to group properties. 

In any case, yes, there are definitely nuances, but the main point is that we need consider the implicit assumptions when using this in other contexts.

Reply:
I don't think it's an apples-to-oranges comparison. It had the same assumptions about how properties were grouped that this does. 

Though I guess, by being about `bssid` it was inherently not making any general claims

Reply:
The bssid was from the time when we were consuming output that included a delimiter between records. The assumptions were different, because we assumed that there was a record separator in the raw output. The raw data was being parsed ('dataflattened') in chunks, where each chunk was known to only contain properties for single wifi network.